### PR TITLE
Littleipsum v1.1.2

### PR DESCRIPTION
**Pull request by stffndtz**
Moved from homebrew-cask to homebrew-versions, see https://github.com/caskroom/homebrew-cask/pull/13269

### CODE REVIEW

File: Casks/littleipsum1.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'littleipsum1' do
+
```
Comment (vitorgalvao) [inline Casks/littleipsum1.rb]: Delete this blank line.

File: Casks/littleipsum1.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  version '1.1.2'
+  sha256 '5fa88e5cd2cc1c837ac3e9ff3a23ce1569835d56c71e2741243ce9f0424099db'
+
+  url "http://littleipsum.com/download/LittleIpsum%20#{version}.zip"
+
```
Comment (vitorgalvao) [inline Casks/littleipsum1.rb]: And this one.

File: Casks/littleipsum1.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  name "LittleIpsum"
+
```
Comment (vitorgalvao) [inline Casks/littleipsum1.rb]: And this one.

File: Casks/littleipsum1.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  homepage 'http://littleipsum.com'
+
```
Comment (vitorgalvao) [inline Casks/littleipsum1.rb]: And this one.

**Comment by fanquake** [comment]
Can you please squash your commits

**Comment by stffndtz** [comment]
will take care if it later today, I don't have any experience with squashing commits. I'll get back asap.

**Pull request merged**